src/assets/images/icons/clock.svg

```diff
@@ -2,6 +2,19 @@
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
  width="16px" height="16px">
-<path fill-rule="evenodd"  fill="rgb(70, 176, 46)"
- d="M8.000,16.000 C3.600,16.000 -0.000,12.400 -0.000,8.000 C-0.000,3.600 3.600,-0.000 8.000,-0.000 C12.400,-0.000 16.000,3.600 16.000,8.000 C16.000,12.400 12.400,16.000 8.000,16.000 ZM8.000,1.600 C4.480,1.600 1.600,4.480 1.600,8.000 C1.600,11.520 4.480,14.400 8.000,14.400 C11.520,14.400 14.400,11.520 14.400,8.000 C14.400,4.480 11.520,1.600 8.000,1.600 ZM7.200,8.800 L7.200,4.000 L8.400,4.000 L8.400,8.160 L12.000,10.320 L11.360,11.360 L7.200,8.800 Z"/>
+  <path fill-rule="evenodd" fill="rgb(70, 176, 46)" d="M11.13043,8.48694L9.04346,7.30432V4.52173c0-0.55652-0.48694-1.04346-1.04346-1.04346
+    S6.95654,3.96521,6.95654,4.52173v2.78259c0,0.20874,0.06952,0.41742,0.20868,0.6261
+    c-0.06958,0.34784,0.1391,0.76526,0.48694,0.97394l2.43481,1.3913c0.48694,0.27826,1.11304,0.1391,1.3913-0.41742
+    C11.82611,9.3913,11.61737,8.7652,11.13043,8.48694z" />
+  <path fill-rule="evenodd" fill="rgb(70, 176, 46)" d="M13.94781,2.64349c-0.20868-0.20874-0.41736-0.41742-0.6261-0.6261C11.93042,0.7652,10.05219,0.06958,8.17389,0
+    H7.82605C5.94781,0.06958,4.06958,0.7652,2.67828,2.0174c-0.20874,0.20868-0.41742,0.41736-0.6261,0.6261
+    c-1.2522,1.3913-1.94781,3.26953-2.0174,5.14783c0,0.06952,0,0.06952,0,0.06952c0,0.06958,0,0.06958,0,0.06958v0.13916
+    c0,0.06958,0,0.06958,0,0.06958c0,0.06952,0,0.06952,0,0.06952c0.06958,1.8783,0.7652,3.75653,2.0174,5.14783
+    c0.20868,0.20874,0.41736,0.41742,0.6261,0.6261C4.06958,15.2348,5.87823,15.93042,7.75653,16h0.06952h0.20874h0.1391h0.06958
+    c1.87823-0.06958,3.68695-0.7652,5.07825-2.0174c0.20874-0.20868,0.41742-0.41736,0.6261-0.6261
+    c1.2522-1.3913,1.94781-3.20001,2.0174-5.07825c0-0.06958,0-0.06958,0-0.06958c0-0.06952,0-0.06952,0-0.06952
+    c0-0.06958,0-0.06958,0-0.06958V7.93042c0-0.06958,0-0.06958,0-0.06958c0-0.06952,0-0.06952,0-0.06952
+    c0-0.06958,0-0.06958,0-0.06958C15.89563,5.84351,15.20001,4.03479,13.94781,2.64349z M7.96521,13.91302
+    c-3.26959,0-5.91302-2.64343-5.91302-5.91302s2.64343-5.91302,5.91302-5.91302S13.87823,4.73041,13.87823,8
+    S11.2348,13.91302,7.96521,13.91302z" />
```

Comment:
if this one is the same as the `became-active` one apart from the color, it's better if we keep just one and strip the color from it and then apply the color we need in the confirmation and delegate monitor cases 🤔

Reply:
will be done in a separate PR as discussed